modules/internal/ChapelDomain.chpl

```diff
@@ -25,6 +25,20 @@ module ChapelDomain {
   public use ChapelBase;
   use ArrayViewRankChange;
 
+  // Fractional value that specifies how full this domain can be
+  // before requesting additional memory. The default value of
+  // 0.5 means that the map will not resize until the map is more
+  // than 50% full. The acceptable values for this argument are
+  // between 0 and 1, exclusive, meaning (0,1). A lower
+  // `defaultHashTableResizeThreshold` value can potentially
+  // improve indexing performance, since the table will likely
+  // have less collisions, while a higher value can help save
+  // memory.
+  // Note that this value also impacts all of Chapel's hash-based
+  // data structures, such as `set` and `map`.
```

Comment:
Does this need to be a multi-line comment to get grabbed?

Reply:
> which may not be the best workflow

On a Mac, my best practice is:
* `make docs`
* `open doc/html/index.html` (or your specific page)
* edit based on what I see
* repeat as necessary (building the chpldocs should be faster on subsequent builds) 
* complain to dyno team about the time required to re-build the dyno docs on each subsequent build